files/__addonLocation__/.prettierignore

```diff
@@ -0,0 +1,9 @@
+# unconventional js
+/blueprints/*/files/
+
+# compiled output
+/dist/
+/declarations/
```

Comment:
yes, because the package has its own lint configs.

Reply:
But we have prettier as a devDep of the root package.json, and also a prettierrc there. 

I think it kinda makes sense to assume you only want one global prettier setup for the whole monorepo. E.g. in case you add multiple addons. Although the test-app has its own, but I guess that's more of a side-effect of generating it with its own blueprint rather than an explicit choice. For eslint this is a bit different I think, as there can be good reasons to have different configs per workspace. Although if you want to argue towards local prettier config that would be legit of course, but then I don't see a reason to still have basically the same file _also_ at the root level!?

As this is a bit orthogonal to the actual change here, does it make sense to put `/declarations/` into the root one _for now_, and tackle this as a separate PR (if you feel this should happen)? As I don't want to block the more important change here...

Reply:
maybe, but when generating with `--addon-only` (what I used to [test with](https://github.com/embroider-build/addon-blueprint/pull/136#issuecomment-1620122134)), the root of anything doesn't come in to play -- so in my testing the change here was needed to have prettier behave properly.

while prettier _is_ in the root package.json, (at least via CI / CLI commands), it is not used as we defer to each sub-package already.

So, if anything, I think we may have a gap in our prettier setup already due to only editors using the root prettier config and we haven't set anything up to lint with it.
I had originally thought (in replying to your comment) about _removing_ prettier (and the ignore file) from the root package.json and in be its own separate PR -- which... I originally was going to fight against changing anything in this PR -- because I thought that the root prettier isn't used at all and was potentially misconfiguration -- but editors will care. But now, after relaizing that it's probably editors that only use the root config and root ignore file and thus only _happens_ to propagate to the nested workspaces, prettier at the root (and the ignore) needs to stay.


As I was reviewing the generated output of the blueprint, something even more alarming popped up! We don't even have prettier configured to run at all! (not via lint / lint:fix or in C.I!)
So, that then needs to be fixed in a separate PR (which I will provide -- tho, I don't think we should block this PR because of this -- having multiple prettier configs / ignores is well supported regardless of how prettier is ran)

Reply:
I have opened https://github.com/embroider-build/addon-blueprint/pull/138 to explore _an option_ for running prettier 
(for small monorepos I don't really have strong opinions on how prettier is ran)

Reply:
Thanks for the investigation. I approved the PR, and yeah, let's look into the prettier situation separately and not block this anymore...